shared/domains/gi/db.js

```diff
@@ -37,9 +37,15 @@ export default (sbp('sbp/selectors/register', {
   'gi.db/set': function (key: string, value: string): Promise<void> {
     return Promise.resolve(sbp('okTurtles.data/set', key, value))
   },
+  'gi.db/delete': function (key: string) {
+    return Promise.resolve(sbp('okTurtles.data/delete', key))
+  },
   'gi.db/log/logHEAD': function (contractID: string): string {
     return `${contractID}-HEAD`
   },
+  'gi.db/log/isLogHEAD': function (key: string): string | boolean {
+    return key.slice(-5) === '-HEAD' ? key.slice(0, -5) : false
+  },
```

Comment:
I think it does make sense, because of separation of concerns (see also my reply to @snowteamer, as I'm not entirely happy with having this selector). If this were done on the frontend, then the frontend would have to have some logic about what a HEAD key looks like, which is determined by `'gi.db/log/logHEAD`. So, we would have logic for the same thing spread across different places.